packages/kbn-plugin-generator/sao_template/template/public/components/app.tsx

```diff
@@ -89,19 +93,20 @@ export const <%= upperCamelCaseName %>App = ({ basename, notifications, http, na
                     <h2>
                       <FormattedMessage
                         id="<%= camelCase(name) %>.congratulationsTitle"
-                        defaultMessage="Congratulations"
+                        defaultMessage="Congratulations, you have successfully created your first Kibana Plugin!"
                       />
                     </h2>
                   </EuiTitle>
                 </EuiPageContentHeader>
                 <EuiPageContentBody>
                   <EuiText>
-                    <h3>
+                    <p>
                       <FormattedMessage
-                        id="<%= camelCase(name) %>.congratulationsText"
-                        defaultMessage="You have successfully created your first Kibana Plugin!"
+                        id="<%= camelCase(name) %>.content"
+                        defaultMessage="Look through the generated code and checkout the plugin developmment documentation."
```

Comment:
If possible, I believe we would want to point them to https://www.elastic.co/guide/en/kibana/current/plugin-development.html

@gchaps or @KOTungseth do we account for the version in some way when adding docs links to the UI? I'm thinking there might be some kind of helper in Kibana for this, but I might also be imagining things :)

Reply:
For reference, you can see the proposed UI copy and link placement here: https://github.com/elastic/kibana/pull/55281#issuecomment-576833494

Reply:
@ryankeairns, @lizozom  we do account for version.  I don't know how it is done, but we use links elsewhere in the UI.

I'm not sure how well "Fetch data" will do with localization.  Can the button text be "Get data" instead?

Also, how about shortening the title line a bit:

Congratulations, you created a Kibana plugin!

Reply:
`Congratulations, you created a new Kibana plugin!` works?

Reply:
Ok on text changes
I'd like to leave the doc link change to a follow up PR, if you don't mind?